tests/FillingTheDishwasher.tex

```diff
@@ -0,0 +1,67 @@
+\section{Filling the Dishwasher [DSPL \& OPL]}
+The robot helps by placing all dirty dishes inside the dishwasher
+
+\subsection{Goal}
+The robot has to correctly identify and manipulate cutlery, kitchenware, and tableware, taking them from the sink and safely placing them in the dishwasher rack.
+
+\subsection{Focus}
+This test focuses on object recognition and manipulation, paying particular attention in orientating objects for an adequate placement in narrow spaces.
+
+\subsection{Setup}
+\begin{enumerate}
+	\item \textbf{Dishwasher:} The dishwasher has the door open and its bottom rack half way outside, ready to be filled in.
+	\item \textbf{Location:} This test takes place in the kitchen of the arena. 
+	\item \textbf{Sink:} The sink has 10 utensils (See \ref{rule:tableware_kitchenware}) in it. There are at least two dishes (or bowls) two mugs (or glasses, or cups), one cooking pan (or pot), and one spoon (or any other cutlery).
+	\item \textbf{Start position:} The robot will start somewhere between the \textit{sink} and the \textit{dishwasher}.
+\end{enumerate}
+
+
+\subsection{Task}
+The robot must execute the following tasks in any order as long as there are utensils in the sink.
+\begin{itemize}
+	\item \textbf{Place cutlery:} The robot takes any cutlery from the sink and safely places it in the cutlery basket of the dishwasher.
+	\item \textbf{Place cooking pans:} The robot takes any cooking pan (or pot, or utensil) from the sink and safely places it in the the dishwasher.
```

Comment:
Same for the pans: leave them in their usage locations, (the stove) and try to lay out the kitchen to have the dishwasher and stove close together to not waste time navigating

Reply:
For simplicity, I would like to have everything in the same pace (the sink , which can be a table), to prevent the robot going here and there. Your suggestion makes sense, and I think changing *sink* to *stove* is no big deal.